app/src/main/java/org/mozilla/fenix/settings/address/view/AddressEditorView.kt

```diff
@@ -103,4 +147,32 @@ class AddressEditorView(
             create()
         }.show()
     }
+
+    private fun bindCountryDropdown() {
+        val adapter = ArrayAdapter<String>(
+            binding.root.context,
+            android.R.layout.simple_spinner_dropdown_item,
+        )
+
+        var selectedPosition = -1
+        var regionPosition = -1
+        var defaultPosition = -1
```

Comment:
I believe we can simplify this by having a getter method that returns the `selectedCountryKey` based on the editing `address`, `region` or the fallback to the `defaultCountry`. That way we are just looking for 1 position instead of 3 in this for loop and the long chain to get the right selection.

Reply:
Yeah good suggestion, thanks